examples/no_host_key_verification.py

```diff
@@ -0,0 +1,5 @@
+from paramiko import client
+
+ssh_client = client.SSHClient()
+ssh_client.set_missing_host_key_policy(client.AutoAddPolicy)
+ssh_client.set_missing_host_key_policy(client.WarningPolicy)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Paramiko call with policy set to automatically trust the unknown host key. https://bandit.readthedocs.io/en/latest/plugins/b507_ssh_no_host_key_verification.html